.devcontainer/README.md

```diff
@@ -140,6 +140,16 @@ If `rush update` fails during post-create:
 2. Try running `rush update` manually after the container starts
 3. Check the Rush logs in `common/temp/`
 
+### GitHub Codespaces Prebuild Failure (Error 1300)
```

Comment:
This section should not be necessary - it's unlikely that we will see regressions, or that other contributors should encounter the same error, after we apply the fix.